services/core-web/common/components/SteppedForm.js

```diff
@@ -10,7 +10,7 @@ import { indexOf } from "lodash";
 const { Item } = Menu;
 
 const propTypes = {
-  children: PropTypes.arrayOf(PropTypes.node).isRequired,
+  children: PropTypes.arrayOf(PropTypes.any).isRequired,
```

Comment:
For some reason I could not figure out, `PropTypes.node` would not recognize the `children` property type correctly, hence this unfortunate change

Reply:
Weird!  Well we can convert it to TS eventually and use an interface.